test/functional/openapi_routes.yaml

```diff
@@ -0,0 +1,5 @@
+petstore-api:
```

Comment:
This file should be in the directory or subdirectory of the test that uses it. It's ok if there are multiple copies of the same file.